crates/bevy_reflect/src/lib.rs

```diff
@@ -549,6 +549,8 @@ mod reflect;
 mod reflectable;
 mod remote;
 mod set;
+mod short_names;
```

Comment:
Minor nit: why not just "short_name" (i.e. singular)? I feel like that reads a little better: `short_name::ShortName`.

Reply:
Ah, looks like that was the original module name. I think we can rename it here.